.cirrus.yml

```diff
@@ -97,7 +102,7 @@ task:
       env:
         BUILD_FULL: ON
         BUILD_SHARED: OFF
-        PLUGINS: ALL;-curlget # The curlget tests fail: https://github.com/ElektraInitiative/libelektra/issues/3382
+        PLUGINS: ALL;-curlget;-xfconf # The curlget tests fail: https://github.com/ElektraInitiative/libelektra/issues/3382 xfce4 is uncommon on macOS
```

Comment:
Why is the plugin disabled here?

Reply:
Because xfce4 is uncommon on macOS as mentioned by you:

> > I could exclude the xfconf-plugin from the cirrus file in the macOS images
> 
> Sounds sensible. Using XFCE under macOS is probably a rare thing. Without actually testing if the macports xfce even works it would be a waste of (CPU) time doing regression checks.
> 
> > FreeBSD
> 
> Sounds like linker path is incomplete. Where are these files (xfconf-0*.so etc.)?

I also included a short description why in the readme with the commit 0f5f8544c42fd3cbc8946a18b1834d6725973c80